post_processing/pylbo/visualisation/spectra/spectrum_merged.py

```diff
@@ -38,13 +38,24 @@ def __init__(self, data, figsize, custom_figure, interactive, legend, **kwargs):
         )
         self.data = data
         self.leg_handle = LegendHandler(interactive)
-        super()._set_plot_properties(kwargs)
+        kwargs_plt = dict(kwargs)
+        kwargs_plt.pop("colors", None)
+        kwargs_plt.pop("color_parameter", None)
+        super()._set_plot_properties(kwargs_plt)
```

Comment:
This is confusing, why not just keep this?

Reply:
Because set_plot_properties needs a dictionary with specific keys, so I can't pass the original kwargs to that function. Added a function in the toolbox to take care of this, that should be more clear.